diagnostic.md

```diff
@@ -32,23 +35,24 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
-
+Getter: .class, :first-child
```

Comment:
Getters are used for getting the values inside these elements, something like `.val()` or `.html()`